### PR TITLE
bump various devex components to golang 1.13

### PR DESCRIPTION
**Pull request by gabemontero**
/assign @adambkaplan 

@openshift/openshift-team-developer-experience FYI

### CODE REVIEW

File: ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ images:
   inputs:
     bin:
       as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
+      - registry.svc.ci.openshift.org/ocp/builder:golang-1.13
```
Comment (gabemontero) [inline ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml]: I wonder if the build change has to merge first before we can do this.

Let's see how the rehearsal jobs go, then circle back.

Comment (gabemontero) [reply ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-master.yaml]: the rehearsal jobs finished ... not sure they covered this permutation

**Comment by adambkaplan** [comment]
@gabemontero rehearsals did nothing:

```
time="2020-02-03T18:08:41Z" level=info msg="Would rehearse too many jobs, will not proceed" pr=6995 rehearsal-jobs=36 rehearsal-threshold=35
```

Let's scale this PR down to individual components which we believe are ready for golang 1.13. Note that any such component that uses alpha-build-machinery needs a recent bump of library-go.

**Comment by gabemontero** [comment]
> @gabemontero rehearsals did nothing:
> 
> ```
> time="2020-02-03T18:08:41Z" level=info msg="Would rehearse too many jobs, will not proceed" pr=6995 rehearsal-jobs=36 rehearsal-threshold=35
> ```
> 
> Let's scale this PR down to individual components which we believe are ready for golang 1.13. Note that any such component that uses alpha-build-machinery needs a recent bump of library-go.

Using library-go/alpha-build-machinery as the decision point, only the builder repo does not have that dependency at this time.

**Comment by gabemontero** [comment]
reduction pushed @adambkaplan

**Comment by gabemontero** [comment]
ok we have clean rehearsahs, include e2e-aws-builds and e2e-aws-image-ecosystem @adambkaplan

File: ci-operator/config/openshift/builder/openshift-builder-master.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.12
+    tag: golang-1.13
```
Comment (adambkaplan) [inline ci-operator/config/openshift/builder/openshift-builder-master.yaml]: Please add this to the 4.4, 4.5, and 4.6 versions of this file.

Comment (gabemontero) [reply ci-operator/config/openshift/builder/openshift-builder-master.yaml]: additional versions changed @adambkaplan

**Pull request merged**